### ISSUE TITLE
[Bug] @qq.com email does not synchronize correctly v1.7.1

### ISSUE THREAD

**Issue by MilkTeaCat52** [bkaankose/Wino-Mail#201]
**Describe the bug**
After the @qq.com email is added for the first time via IMAP, the folders show up successfully. However, synchronization never succeeds. The error message is: The IMAP server replied to the 'FETCH' command with a 'Bad' response: UID parameters! 

Mail cannot be sent. When deleting a draft, an error appears, showing an error about a UUID.

**Expected behavior**
Synchronization of emails from the Tencent IMAP server.

**Screenshots**
![image](https://github.com/bkaankose/Wino-Mail/assets/91018175/6c4b6f51-bccb-45b8-b014-201de584010e)
![image](https://github.com/bkaankose/Wino-Mail/assets/91018175/f2e3376e-55a0-4c1b-835b-0c7f6f46cbd5)

**Additional context**
I followed the guide provided by Tencent to set up IMAP account:
https://service.mail.qq.com/detail/0/339

**Comment by twinstar6980** [comment]
me too, qq mail worked fine in previous wino versions, but after I updated wino to version 1.7, it reported the same error.

**Comment by CanghaiDragon** [comment]
I have the similar question, I can sign in but my inbox is just empty.

**Comment by MilkTeaCat52** [comment]
[ImapProtocolLog.log](https://github.com/user-attachments/files/16195068/ImapProtocolLog.log)
[Logs.txt](https://github.com/user-attachments/files/16195069/Logs.txt)

These are some logs from Wino related to this issue.

**Comment by bkaankose** [comment]
Is this QQ server available for everyone? If so, could you please create a temporary account for me and pass the credentials on Discord?

**Comment by MilkTeaCat52** [comment]
The QQ server should be available for everyone (it's one of the largest email providers in China). However, I was not able to create a temporary account due to "security issues" with my phone number.

I'll be able to create an account in 2 weeks, but if anyone wants to try in the meantime, the website is mail.qq.com

**Comment by wmjlxl** [comment]
i upgraded to Unlimited Wino Mail today and found out that it seems QQ mail didnt work.

hope to fix this issue asap.

thanks a lot.

**Comment by bkaankose** [comment]
@MilkTeaCat52 can I close this issue as fixed?

**Comment by MilkTeaCat52** [comment]
Yeah, this is better in the newer versions. There're still some problems with sync but I think they would be better addressed in another issue?

**Comment by bkaankose** [comment]
> Yeah, this is better in the newer versions. There're still some problems with sync but I think they would be better addressed in another issue?

Latest beta release (NET9 Upgrade) has imap improvements. You can give that a try and open a new issue if you still have any.
